### PR TITLE
Rename linuxkit/mkimage to linuxkit/mkimage-aws

### PR DESCRIPTION
**Pull request by deitch**
Signed-off-by: Avi Deitcher <avi@deitcher.net>

**- What I did**
Renamed the image for building an aws image from `linuxkit/mkimage` to `linuxkit/mkimage-raw`

As discussed with @justincormack , this isn't really a "raw" image, but a very AWS-specific one. There already are PRs that will provide `mkimage-raw-efi` (multi-platform) and `mkimage-raw-bios` (smallest possible size).

Also switches from old `Makefile` using `package.mk` to `linuxkit pkg ...`

**- How I did it**
Changed the files

**- How to verify it**
Run it

**- Description for the changelog**

Rename `linuxkit/mkimage` to `linuxkit/mkimage-aws`

Some notes:

* Not actually "renaming", but keeping original around for backwards compatibility. All future pushes will be the other image.
* Once this is done, a `moby/tool` PR will be opened to change the format name and point to the newer image

**- A picture of a cute animal (not mandatory but encouraged)**
hear no evil

### CODE REVIEW

**Comment by deitch** [comment]
FYI, unsure if the renaming will break tests. Will let full CI run as usual.

File: pkg/mkimage-aws/Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+IMAGE=mkimage
```
Comment (justincormack) [inline pkg/mkimage-aws/Makefile]: this should be `mkimage-aws` now

Comment (justincormack) [reply pkg/mkimage-aws/Makefile]: actually, this file should be deleted now!

Comment (deitch) [reply pkg/mkimage-aws/Makefile]: That’s what I get for doing PRs late at night! I’ll update shortly.

Comment (deitch) [reply pkg/mkimage-aws/Makefile]: Pushing...

**Comment by deitch** [comment]
Ugh, the merge of #2578 messed this one up, and rebase isn't working cleanly. Going to try and redo the branch. If I cannot, will replace with a new PR.

**Comment by deitch** [comment]
Well, it worked. Go figure.

File: pkg/mkimage-aws/Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.PHONY: push forcepush tag forcetag show-tag
```
Comment (ijc) [inline pkg/mkimage-aws/Makefile]: See https://github.com/linuxkit/linuxkit/pull/2558#discussion_r143764152

**Comment by deitch** [comment]
@justincormack where do we stand on this one? Can we merge it in. and then proceed adding the 2 formats and renaming the 1 in `moby/tool`? I will open that PR as soon as this is in.

**Comment by deitch** [comment]
I am closing this one out. I forgot that we use `mkimage` for `qcow` as well as `aws`. We should do something with both of them, but for now, renaming to `mkimage-aws` does not make sense.

**Pull request closed without merging**